src/common/snippets/src/pass/collapse_subgraph.cpp

```diff
@@ -502,16 +563,21 @@ TokenizeSnippets::TokenizeSnippets() {
         if (body_results.size() != subgraph_result_inputs.size()) {
             throw ngraph_error("body results and node results size mismatch during subgraph collaps");
         }
+
+        auto body = op::create_body(node->get_friendly_name(), body_results, body_parameters);
+        const auto hidden_non_scalar_constant_count = get_potentional_non_scalar_constants_count(body);
+
```

Comment:
I have added the separate parameter `m_non_scalar_constants_count` for that. So now we shouldn't calculate non-scalar Constants for each FakeQuantize during tokenization. Now we should just summarize these parameters of all subgraphs which will be collapsed. I hope it will help with load network time as well.